Dockerfile.d/test-integration-rootless.sh

```diff
@@ -21,5 +21,26 @@ if [[ "$(id -u)" = "0" ]]; then
 else
 	containerd-rootless-setuptool.sh install
 	containerd-rootless-setuptool.sh install-buildkit
+        containerd-rootless-setuptool.sh install-stargz
+        cat <<EOF > /home/rootless/.config/containerd/config.toml
+[proxy_plugins]
+  [proxy_plugins."stargz"]
+    type = "snapshot"
+    address = "/run/user/1000/containerd-stargz-grpc/containerd-stargz-grpc.sock"
+EOF
+        systemctl --user restart containerd.service
+        sleep 3
+        if [ -d "${HOME}/.ipfs" ] ; then
+            echo "IPFS has already been set up."
+        else
+            export IPFS_PATH=${HOME}/.ipfs
+            mkdir -p ${IPFS_PATH}
+            nerdctl run -d --name ipfs_host -v ${IPFS_PATH}:/data/ipfs -p 127.0.0.1:5001:5001 \
+                    ghcr.io/stargz-containers/ipfs/go-ipfs:v0.10.0-org  daemon --offline
+            sleep 3
+            echo "ipfs = true" > ${HOME}/.config/containerd-stargz-grpc/config.toml
+            systemctl --user restart stargz-snapshotter.service
+            sleep 3
+        fi
```

Comment:
The logic looks too much complicated.

Maybe we should just run IPFS as a systemd service?

Reply:
Rootless nerdctl (inside rootlesskit network namespace) can't access to IPFS daemon API (127.0.0.1:5001) on the host?

Reply:
Right. I thought we could just use a public IP.

Reply:
Wondering we should have `containerd-rootless-setuptool.sh install-ipfs` to run IPFS daemon in the netns

https://github.com/containerd/nerdctl/blob/5b52e3b2523647c13d41723cb499dbe1b520971a/extras/rootless/containerd-rootless-setuptool.sh#L397-L419

Reply:
SGTM